examples/Eady_growth_rate.py

```diff
@@ -0,0 +1,10 @@
+import metpy.calc as mpcalc
+from metpy.units import units
+import  xarray as xr
+from netCDF4 import Dataset
+import pandas as pd
+
+from metpy.cbook import get_test_data
+data = xr.open_dataset(get_test_data('narr_example.nc', False))
+data = data.metpy.parse_cf().squeeze()
+edr=mpcalc.kinematics.eady_growth_rate(data['Temperature'],data['u_wind'],data['Geopotential_height'],data['lat'])
```

Comment:
I think this example would be better placed in the docstring for the calculation like we do elsewhere, for example:

https://github.com/Unidata/MetPy/blob/09b4df42c9f4bb9a8ccec07a6de7ed88e5a72aa7/src/metpy/calc/kinematics.py#L625-L641